csm_api_client/service/cfs.py

```diff
@@ -663,44 +662,55 @@ def get_suggested_debug_command(self) -> Optional[str]:
             The command that the admin should run to debug the failure,
                 or None if no command is available or necessary.
         """
-        container_execution_order = [
-            'inventory',
-            'ansible-*',
-            'teardown'
-        ]
 
         # This shouldn't happen but protect against it anyway
         if self.pod is None or self.pod.metadata is None:
             return None
 
         kubectl_cmd = f'kubectl logs -n {self.KUBE_NAMESPACE} {self.pod.metadata.name}'
-        if self.failed_init_containers:
+        failed_container = self.get_first_failed_container(
+            self.failed_init_containers,
+            self.failed_containers
+        )
+        if failed_container is None:
+            return None
+
+        return f'{kubectl_cmd} -c {failed_container}'
+
+    @staticmethod
+    def get_first_failed_container(
+        failed_init_containers: list[str],
+        failed_containers: list[str]
+    ) -> Optional[str]:
+        container_execution_order_prefixes = [
```

Comment:
It doesn't seem like it's even necessary for these to be prefixes anymore as I think these are the full names of the containers. But it's probably fine to leave them as prefixes.

Edit: I suppose this is good for backwards compatibility with older versions of CFS that used multiple Ansible containers, one for each layer.

Reply:
Yep, that was my thought. For the new container names, the container name is just the prefix, and for older container names the prefix is the same.